### PR TITLE
Feat/highlighted shows

### PR DESCRIPTION
**Pull request by eriktmidtun**
Made a new component to display and play the three newest episodes from the front page.

### CODE REVIEW

**Comment by eriktmidtun** [comment]
Known bug: Episodes cant be played on phone. Happened after refactoring with serverside rendering from dev.

**Comment by eriktmidtun** [comment]
Bug: episode images get set to full image height on iOS-devices

File: src/components/FrontPage/components/HighlightedShows/index.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+  constructor(props) {
+    super(props);
+    // test state
+    this.state = {
+      selectedShowIndex: 0,
+    };
+  }
+
+  startInterval() {
+      this.interval = setInterval(
```
Comment (santi) [inline src/components/FrontPage/components/HighlightedShows/index.js]: Same comment about binding `this` here, as in `NewestEpisodes` component.

File: src/components/FrontPage/components/HighlightedShows/index.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
+import React from 'react';
```
[the remark this answers is not in the document]
Comment (santi) [reply src/components/FrontPage/components/HighlightedShows/index.js]: Not quite sure, but looks like this is an old version of the `NewestEpisodes` component. Might have been added by mistake?

**Pull request merged**